src/core/ui.js

```diff
@@ -59,14 +59,29 @@ respecPill.addEventListener("click", function(e) {
   this.setAttribute("aria-expanded", String(menu.hidden));
   menu.hidden = !menu.hidden;
 });
+document.documentElement.addEventListener("keyup", e => {
+  e.stopPropagation();
+  if (e.keyCode === 73) {
+    if (menu.hidden) {
+      menu.classList.remove("respec-hidden");
+      menu.classList.add("respec-visible");
+      respecPill.focus();
+    } else {
+      menu.classList.add("respec-hidden");
+      menu.classList.remove("respec-visible");
+    }
+    respecPill.setAttribute("aria-expanded", String(menu.hidden));
```

Comment:
This automatically coerces to a string, I believe (or is TypeScript causing an error here?). 
```suggestion
    respecPill.setAttribute("aria-expanded", menu.hidden);
```

Reply:
TS disallows automatic coercing so probably.